src/alerts/slack.coffee

```diff
@@ -4,9 +4,7 @@ request = require 'request'
 module.exports = class SlackAlert extends Alert
   sendToSlack: (message) ->
     options =
-      url: "#{@config.host}/services/hooks/incoming-webhook"
-      qs:
-        token: @config.token
+      url: "https://hooks.slack.com/services/#{config.token}"
```

Comment:
I like that idea a lot - saves you and me a lot of time if they change the scheme of the URL in the future. :+1: